scripts/prepublish.sh

```diff
@@ -6,7 +6,6 @@ npm run test
 
 npx hardhat export --export-all deployments.json
 
-rm index.ts
```

Comment:
this line was here to make sure we're working with a "blank canvas" when this script starts. Maybe instead of removing the line, we change it to `rm publish/index.ts`?

Reply:
the line below removes the publish folder and the files within.

Reply:
hahaha yes good point!